docs/source/usage_guides/distributed_inference.md

```diff
@@ -15,12 +15,18 @@ rendered properly in your Markdown viewer.
 
 # Distributed Inference with 🤗 Accelerate
 
-Distributed inference is a common use case, especially with natural language processing (NLP) models. Users often want to
-send a number of different prompts, each to a different GPU, and then get the results back. This also has other cases
-outside of just NLP, however for this tutorial we will focus on just this idea of each GPU receiving a different prompt,
-and then returning the results.
+Distributed inference can fall into three brackets:
 
-## The Problem
+1. Loading an entire model onto each GPU and sending chunks of a batch through each model at a time
+2. Load parts of a model onto each GPU and process a single input at one time
```

Comment:
```suggestion
2. Loading parts of a model onto each GPU and processing a single input at one time
```

Reply:
For consistency with the first item in the list, let's say "loading" instead of "load".